src/server/cluster_family_test.cc

```diff
@@ -375,5 +452,21 @@ TEST_F(ClusterFamilyEmulatedTest, ClusterInfo) {
   EXPECT_THAT(cluster_info, HasSubstr("cluster_size:1"));
 }
 
+TEST_F(ClusterFamilyEmulatedTest, ClusterShards) {
+  EXPECT_THAT(Run({"cluster", "shards"}),
+              RespArray(ElementsAre("slots",                                           //
+                                    RespArray(ElementsAre(IntArg(0), IntArg(16383))),  //
+                                    "nodes",                                           //
+                                    RespArray(ElementsAre(                             //
+                                        RespArray(ElementsAre(                         //
+                                            "id", _,                                   //
```

Comment:
maybe compare the id to RunAdmin({"dflycluster", "myid"}) ?

Reply:
That's a good idea, once that command is merged in :)

Reply:
Oh, sorry, it's already merged in. I'll use it.

Reply:
Done, PTAL.